packages/api-v2/src/auth/auth.service.ts

```diff
@@ -16,9 +19,7 @@ export class AuthService {
     private readonly jwtService: JwtService
   ) {}
 
-  /**
-   * Registers a new student in the db and logs the student in.
-   */
+  /** Registers a new student in the db and logs the student in. */
   async register(createStudentDto: CreateStudentDto): Promise<Student> {
     // create a new student
     const newStudent = await this.studentService.create(createStudentDto);
```

Comment:
If `newStudent` is null here, shouldn't we log this instead of just returning `null`?

Reply:
ahh yea, i didn't log here cause we don't know the reason newStudent is null here... we only know the reason in studentService.create which in turn does log. If we logged here all we could do is say that create returned null for some reason which isn't very insightful